### ISSUE TITLE
type error

### ISSUE THREAD

**Issue by Qvatra** [swimmadude66/html-webpack-link-type-plugin#12]
when i try to use this plugin i get:
```
ERROR in   TypeError: _this._transformAssets is not a function
  
  - plugin.js:61 Compilation.<anonymous>
    [bagp]/[html-webpack-link-type-plugin]/dist/plugin.js:61:49
  
  - Tapable.js:272 Compilation.applyPluginsAsyncWaterfall
    [bagp]/[tapable]/lib/Tapable.js:272:13
  
  - index.js:645 
    [bagp]/[html-webpack-plugin]/index.js:645:12
  
  - index.js:151 
    [bagp]/[html-webpack-plugin]/index.js:151:16
```
any help would be appreciated.

this is my plugin section in webpack:
```  plugins: [
    new webpack.DefinePlugin({
      'process.env': config.dev.env
    }),
    new webpack.HotModuleReplacementPlugin(),
    new webpack.NoEmitOnErrorsPlugin(),
    new HtmlWebpackPlugin({
      filename: 'index.html',
      template: 'index.html',
      inject: true,
      minify: {
        collapseWhitespace: true,
        removeComments: true,
        removeRedundantAttributes: true,
        useShortDoctype: true
      }
    }),
    new LinkTypePlugin({
      '**/*.css' : 'text/css'
    }),
    new FriendlyErrorsPlugin()
  ]
```

**Comment by swimmadude66** [comment]
@Qvatra sorry to hear its not working! Can you please paste the lines from your package.json specifying the version of webpack, html-webpack-plugin, and html-webpack-link-type-plugin please?

**Comment by swimmadude66** [comment]
This issue occurs due to a mismatch in webpack/html-webpack-plugin/this plugin versions, but without the version numbers causing your error there's not much I can do. Closing now, and will reopen if more information is provided.

**Comment by lin-credible** [comment]
```
"html-webpack-link-type-plugin": "^1.1.0",
"html-webpack-plugin": "^3.2.0"
```

Same err : (

**Comment by localyost3000** [comment]
@lin-credible whats the webpack version?

**Comment by localyost3000** [comment]
This has been fixed in v1.1.1. I apologize for taking so long on this simple fix, but in my defense, getting a webpack 3 build to work at all has become incredibly difficult. most common plugins are deprecated, and upgrading to v4 is a borderline necessity at this point. That being said, I know not all situations allow for that, so hopefully this helps address those!
